packages/mobile_app/lib/components/prophecy_record.dart

```diff
@@ -8,140 +8,308 @@ import 'package:language/language.dart';
 import 'package:app/theme/app_colors.dart';
 import 'package:users_repository/users_repository.dart';
 
-class ProphecyRecord extends StatelessWidget {
+const _LOW = 5.0;
+const _HIGH = 5.1;
```

Comment:
I suggest renaming to something like `_PROPHECY_LOW` and `_PROPHECY_HIGH` respectively

Reply:
Will be done